tools/cloudera/build_hue_common.sh

```diff
@@ -117,6 +117,12 @@ function redhat8_ppc_install() {
       xmlsec1-openssl \
       libss \
       ncurses-devel'
+    # Ensure pg_config is available
+    export PG_CONFIG=$(which pg_config)
```

Comment:
1. Is `pg_config` only needed at the build time and not runtime?
2. Shouldn't we fail the script early if `pg_config` is not found? It will anyways fail when we attempt to do the `pip install`.
3. Nit: There is too much of repetition across all OS's. Can we abstract out a function, something like `install_psycopg`? If it makes things more complex, then please ignore.

Reply:
1. pg_config is required at only build time for building psycopg2. 
2. added an exit, when psycopg2 is not found. 
3. I have to retest it in all Os'es. I'll create a function to do the psycopg2 install and submit a PR later